src/main/java/net/imagej/ops/stats/StatsNamespace.java

```diff
@@ -90,6 +93,44 @@
 		return result;
 	}
 
+	@SuppressWarnings("rawtypes")
+	@OpMethod(op = net.imagej.ops.stats.IntegralMean.class)
+	public DoubleType integralMean(final DoubleType out,
+		final RectangleNeighborhood in1, final Interval in2)
+	{
+		final DoubleType result = (DoubleType) ops().run(
+			net.imagej.ops.stats.IntegralMean.class, out, in1, in2);
+		return result;
+	}
+
+	@SuppressWarnings("rawtypes")
+	@OpMethod(op = net.imagej.ops.stats.IntegralSum.class)
+	public DoubleType integralSum(final RectangleNeighborhood in) {
+		final DoubleType result = (DoubleType) ops().run(
+			net.imagej.ops.stats.IntegralSum.class, in);
+		return result;
+	}
+
+	@SuppressWarnings("rawtypes")
+	@OpMethod(op = net.imagej.ops.stats.IntegralSum.class)
+	public DoubleType integralSum(final DoubleType out,
```

Comment:
why does `Integral` sum need its own method signature? Shouldn't the framework take care about choosing the fastest possible `Op` for a certain use-case?